core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java

```diff
@@ -157,6 +157,12 @@ public void testProcessQuery() throws Exception {
         assertEquals("The variant summary does not match",
             ONCOGENIC_MUTATIONS_DEFAULT_SUMMARY, indicatorQueryResp.getVariantSummary());
 
+        // Check special variant VUS Mutations
+        query = new Query(null, null, null, "BRD4", InferredMutation.VUS_MUTATIONS.getVariant(), null, null, null, null, null, null, null);
+        indicatorQueryResp = IndicatorUtils.processQuery(query, null, false, null);
+        assertTrue("The variantExist in the response is not true, but it should.", indicatorQueryResp.getVariantExist() == true);
```

Comment:
This test is invalid since the BRD4 does not have VUS yet. We could comment out this section and update it when we actually have the data.